docs/schema_script.txt

```diff
@@ -20,4 +20,7 @@ asset.doc.type.update :create true :description "Project metadata" :type tigerda
     :element -name data_users_ro -type string -index true -min-occurs 0               -label "A person who has read-only access privileges to the project" \
     :element -name departments   -type string -index true -min-occurs 1               -label "The primary Princeton University department(s) affiliated with the project" \
     :element -name created_on    -type date               -min-occurs 1 -max-occurs 1 -label "Timestamp project was created" \
-    :element -name created_by    -type string             -min-occurs 1 -max-occurs 1 -label "User that created the project" >
+    :element -name created_by    -type string             -min-occurs 1 -max-occurs 1 -label "User that created the project" \
+    :element -name updated_on    -type date               -min-occurs 0 -max-occurs 1 -label "Timestamp project was updated" \
+    :element -name updated_by    -type string             -min-occurs 0 -max-occurs 1 -label "User that created the created" >
```

Comment:
Is this label right? Maybe it's supposed to read "User that updated the project"?

Reply:
"created the created" -- yeah, of course that's what I meant 🤦🏼 -- I'll fix it 😄